### PR TITLE
apply clippy lints

### PR DESCRIPTION
**Pull request by hellow554**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -352,11 +353,7 @@ fn fetch_master_commit_via_http(
         status => bail!("received status {} for URL {}", status, URL),
     }
     let master_commit = response.text()?;
-    if master_commit.len() == 40
-        && master_commit
-            .chars()
-            .all(|c| '0' <= c && c <= '9' || 'a' <= c && c <= 'f')
-    {
+    if master_commit.len() == 40 && master_commit.chars().all(|c| c.is_ascii_hexdigit()) {
```
Comment (hellow554) [inline src/main.rs:356]: I could see, that you would like to reject upper case chars, in that case I would suggest

```
('0'..='9').contains(c) && ('a'..='f').contains(c)
```

Comment (hellow554) [reply src/main.rs]: done. Nice solution, haven't thought about `matches!` in this case.
Maybe it's worth to tell the clippy people ;)

**Comment by hellow554** [comment]
That's funny! If I do `cargo fmt` that branch stays clean, e.g. there's nothing *my* rustfmt would like to format.
So I would suggest that there should be a project specific rustfmt.toml that has the desired options listed. 

I'm not aware that I have a global rustfmt.toml (e.g. there's no `~/.rustfmt.toml`), but here's my current output.

<details>
<summary>rustfmt --print-config current .</summary>

```
max_width = 100
hard_tabs = false
tab_spaces = 4
newline_style = "Auto"
indent_style = "Block"
use_small_heuristics = "Default"
fn_call_width = 60
attr_fn_like_width = 70
struct_lit_width = 18
struct_variant_width = 35
array_width = 60
chain_width = 60
single_line_if_else_max_width = 50
wrap_comments = false
format_code_in_doc_comments = false
comment_width = 80
normalize_comments = false
normalize_doc_attributes = false
license_template_path = ""
format_strings = false
format_macro_matchers = false
format_macro_bodies = true
empty_item_single_line = true
struct_lit_single_line = true
fn_single_line = false
where_single_line = false
imports_indent = "Block"
imports_layout = "Mixed"
imports_granularity = "Preserve"
group_imports = "Preserve"
reorder_imports = true
reorder_modules = true
reorder_impl_items = false
type_punctuation_density = "Wide"
space_before_colon = false
space_after_colon = true
spaces_around_ranges = false
binop_separator = "Front"
remove_nested_parens = true
combine_control_expr = true
overflow_delimited_expr = false
struct_field_align_threshold = 0
enum_discrim_align_threshold = 0
match_arm_blocks = true
match_arm_leading_pipes = "Never"
force_multiline_blocks = false
fn_args_layout = "Tall"
brace_style = "SameLineWhere"
control_brace_style = "AlwaysSameLine"
trailing_semicolon = true
trailing_comma = "Vertical"
match_block_trailing_comma = false
blank_lines_upper_bound = 1
blank_lines_lower_bound = 0
edition = "2015"
version = "One"
inline_attribute_width = 0
merge_derives = true
use_try_shorthand = false
use_field_init_shorthand = false
force_explicit_abi = true
condense_wildcard_suffixes = false
color = "Auto"
required_version = "1.4.37"
unstable_features = false
disable_all_formatting = false
skip_children = false
hide_parse_errors = false
error_on_line_overflow = false
error_on_unformatted = false
report_todo = "Never"
report_fixme = "Never"
ignore = []
emit_mode = "Files"
make_backup = false
```
</details>

**Comment by kennytm** [comment]
I've got the same diff as Travis. The config is the same. What is your rustc & rustfmt version?

```
$ rustc -V
rustc 1.54.0 (a178d0322 2021-07-26)

$ rustfmt -V
rustfmt 1.4.37-stable (a178d032 2021-07-26)
```

**Comment by hellow554** [comment]
Same:

```
marcel@IDENT0772 /t/t/rustup-toolchain-install-master (master)> rustc -V
rustc 1.54.0 (a178d0322 2021-07-26)
marcel@IDENT0772 /t/t/rustup-toolchain-install-master (master)> rustfmt -V
rustfmt 1.4.37-stable (a178d03 2021-07-26)
```

**Pull request merged**